src/OpenRoad.cc

```diff
@@ -237,6 +239,8 @@ OpenRoad::init(Tcl_Interp *tcl_interp)
   initLogger(logger_, tcl_interp);
   initGui(this); // first so we can register our sink with the logger
   Odbtcl_Init(tcl_interp);
+  Upf_Init(tcl_interp);
+  evalTclInit(tcl_interp, sta::upf_tcl_inits);
```

Comment:
Should this be a tool on it's own? ie. with a `MakeUPF()`

Reply:
I told Ahmad he could fix this in another PR as we already have several cases here that need fixing.